importance_sampling_optimizer/config.py

```diff
@@ -4,7 +4,8 @@
 SLEEP_TIME = 60
 
 JOB_TEMPLATE_IMP_SAMPLING = {
-    'input': ['eos:/eos/experiment/ship/skygrid/importance_sampling'],
+    'input': ['eos:/eos/experiment/ship/skygrid/importance_sampling',
+              'eos:/eos/ship/data/Mbias'],
```

Comment:
1. We don't want to copy all the files in there! Most of the are pretty large and not needed.
2. It's `/eos/experiment/ship`, not `/eos/ship`
3. @sashabaranov : Does the `eos` loader use `eospublic.cern.ch` or still `eoslhcb.cern.ch`?

Reply:
@olantwin 

1. Yes, we dont want this.
2. For production output we use `eos:/eos/experiment/ship/skygrid/background-prod-2018/$JOB_ID`, so maybe it is right
3. it uses root://eospublic.cern.ch/

Reply:
Thanks for the quick reply!
1. So maybe we should copy the file to the cluster once and then access it locally? Any other suggestions?
2. `/eos/experiment` is right. This was concerning `eos:/eos/ship/data/Mbias` which won't work
3. Perfect :)

Reply:
> So maybe we should copy the file to the cluster once and then access it locally? Any other suggestions?

If you'll need to copy something — just tell me what.

Reply:
I'd say we need `root://eospublic.cern.ch//eos/experiment/ship/data/Mbias/pythia8_Geant4-withCharm_onlyMuons_4magTarget.root` somewhere where it can be easily be accessed by the docker workers. Is that possible? Maybe where we'd normally keep the input files for the workers?